styles/styles.css

```diff
@@ -267,12 +267,87 @@ blockquote {
   margin: var(--spacer-element-05) 0 var(--spacer-element-05);
 }
 
-main .columns-gap-50>div {
+main .column-content-top-border div>div {
+  margin-bottom: 32px;
+  max-width: 350px;
+}
+
+main .columns.columns-gap-50>div {
   gap: 50px
 }
 
-main .column-content-top-border div>div {
-  margin-bottom: 32px
+main .padding-bottom-40 {
+  padding-bottom: 40px;
+}
+
+main .padding-bottom-60 {
+  padding-bottom: 60px;
+}
+
+main .margin-bottom-40 {
+  margin-bottom: 40px;
+}
+
+main .margin-bottom-60 {
+  margin-bottom: 60px;
+}
+
+main .section.margin-top-60,
+main .section .columns.margin-top-60 {
+  margin-top: 60px;
+}
+
+main .section.margin-top-40,
+main .section .columns.margin-top-40 {
+  margin-top: 40px;
+}
+
+main .bg-neutral-bone {
+  background-color: var(--neutral-bone);
+}
+
+main .section.notification-section {
+  padding-top: 14px;
+  padding-bottom: 14px;
+}
+
+main .section.notification-section>div>p::before {
+  content: '';
+  display: inline-block;
+  width: 5px;
+  height: 32px;
+  margin: -9px 50px -7px 0; 
+  background-image: var(--gradient-diagonal-down);
+}
+
+main .section.what-we-do>.default-content-wrapper>h4 {
```

Comment:
Now we're getting very specific. This style will noly ever be used with `what-we-do` section?